ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactTextInputManager.java

```diff
@@ -172,6 +174,40 @@ public void setFontSize(ReactEditText view, float fontSize) {
         (int) Math.ceil(PixelUtil.toPixelFromSP(fontSize)));
   }
 
+  @ReactProp(name = ViewProps.FONT_FAMILY)
+  public void setFontFamily(ReactEditText view, String fontFamily) {
+    view.setTypeface(Typeface.create(fontFamily,view.getTypeface().getStyle()));
+  }
+
+  @ReactProp(name = ViewProps.FONT_WEIGHT)
+  public void setFontWeight(ReactEditText view, @Nullable String fontWeightString) {
```

Comment:
Yeah, good call. Moving this to a shared place would be better but honestly  I have no idea where that place would be. If you have any suggestion about it, I'm happy to do it. In the meantime, I'm adding a comment.